packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -3,20 +3,20 @@
   <header class="relative">
     {{#askMobileStatus}}
       {{#needsMobileConfirmed}}
-        <button id="back-btn" class="me-4 tablet:me-0 tablet:p-4 inline-block tablet:absolute tablet:-start-20 -top-1.5 w-5 h-3.5 p-5 bg-back-arrow bg-auto bg-center bg-no-repeat rtl:transform rtl:-scale-x-100" aria-label="{{#t}}Back{{/t}}" title="{{#t}}Back{{/t}}"></button>
+        <button id="back-btn" class="me-4 tablet:me-0 tablet:p-4 inline-block tablet:absolute tablet:-start-20 -top-1.5 w-5 h-3.5 p-5 bg-back-arrow bg-auto bg-center bg-no-repeat rtl:transform rtl:-scale-x-100 rounded hover:bg-grey-50 active:bg-grey-100 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500" aria-label="{{#t}}Back{{/t}}" title="{{#t}}Back{{/t}}"></button>
```

Comment:
Eduardo approved!

I tried to apply these same classes to the back button in Settings, but the DOM is set up a little differently over there and it was overlapping with the `.card`. We could probably use a 1-pointer to apply a similar style or maybe I'll circle back here and see if it's pretty quick to better align these.


https://github.com/mozilla/fxa/assets/13018240/16af761c-2016-4dfe-a3c1-5fae32aa8fc5